packages/mattermost-redux/src/types/actions.ts

```diff
@@ -1,30 +1,32 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {ThunkAction} from 'redux-thunk';
 import {AnyAction} from 'redux';
 import {BatchAction} from 'redux-batched-actions';
 
 import {GlobalState} from '@mattermost/types/store';
+import {ServerError} from '@mattermost/types/errors';
 
 export type GetStateFunc = () => GlobalState;
 export type GenericAction = AnyAction;
 export type Thunk = (b: DispatchFunc, a: GetStateFunc) => Promise<ActionResult> | ActionResult;
 
 export type Action = GenericAction | Thunk | BatchAction | ActionFunc;
 
-export type ActionResult<Data = any, Error = any> = {
+export type ActionResult<Data = unknown, Error = unknown> = {
```

Comment:
Should `Error` default to `ServerError` instead of `unknown`? I also wonder if making `Data` default to something like `never` so that we stop having to return `{data: true}` everywhere would improve things or not